### ISSUE TITLE
Make the Grid a Parallax Layer

### ISSUE THREAD

**Issue by mango-gremlin** [mango-gremlin/arch-enemies#120]
## Motivation
Makes the grid cover the entire screen dynamically.

## Implementation
Use a Parallax Layer to draw the grid: 
https://docs.godotengine.org/en/stable/classes/class_parallaxlayer.html

**Comment by koennixer** [comment]
I do not fully understand what the concept here is. A parallax layer cannot have a tile map as a child, that simply doesn't work. Currently we are using mono colored tile sets as the background, I assume we will change that and once we do, we will most likely use a parallax background for the actual backgrounds, when we do that adding a grid on top of that as a parallax layer will be a nightmare, because it will attempt to shift with the camera. I feel like the current solution, although clunky, works while not being a hazard and can easily be adapted for larger or different levels.

In short I don't see why a parallax backgrounds, who's entire purpose is to shift around with the camera, is helpful for a grid, which is supposed to be static even when the camera, or anything for that matter, moves.

**Comment by mango-gremlin** [comment]
Let me rephrase this, I wasn't aware a tilemap cannot be contained in a parallax layer, but in any case, that doesn't really have anything to to with the idea itself. What i meant was that we should contain the two grid tile variations in a parallax layer, in any way that you can do it in godot. I will edit the issue and delete the word tilemap.
I know the current solution works, but like you said, it is clunky, and it doesn't take much work to implement the parallax layer. It would save us a bit of time on new levels, and more importantly, it is a much cleaner solution. I don't see any reason not to do it.

Additionally, we as a group already agreed on doing this, so I don't really see the point of discussion. This is something that can be done fairly quickly, and will probably only be done if there is time at the end.

**Comment by mango-gremlin** [comment]
due to implementing #181 this will no longer be necessary
